app/policies/ticket_policy.rb

```
@@ -0,0 +1,88 @@
+# app/policies/ticket_policy.rb
+class TicketPolicy < ApplicationPolicy
+  attr_reader :user, :ticket
+
+  def initialize(user, ticket)
+    @user = user
+    @ticket = ticket
+  end
```

RuboCop (1.75.5)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)